onepassword/data_source_onepassword_item_test.go

```diff
@@ -95,6 +95,31 @@ func compareItemToSource(t *testing.T, dataSourceData *schema.ResourceData, item
 	}
 	compareStringSlice(t, getTags(dataSourceData), item.Tags)
 
+	for _, section := range item.Sections {
+		t.Errorf("Missing Implementation for %s", section.Label)
+		//keyedSection := dataSourceData.Get(fmt.Sprintf("keyed_sections.%s", section.Label)).(map[string]interface{})
+		//dataSourceData.GetRawState()
+		//if keyedSection == nil {
+		//	t.Errorf("Expected keyed section %v to exist", section.Label)
+		//}
+		//if keyedSection["id"] != section.ID {
+		//	t.Errorf("Expected keyed section %v to have id %v got %v", section.Label, section.ID, keyedSection["id"])
+		//}
+		//
+		//for _, field := range item.Fields {
+		//	if field.Section != nil && field.Section.ID == section.ID {
+		//		keyedField := dataSourceData.Get(fmt.Sprintf("keyed_sections.%s.keyed_fields.%s", section.Label, field.Label)).(map[string]interface{})
+		//
+		//		if keyedField == nil {
+		//			t.Errorf("Expected keyed field %v to exist", field.Label)
+		//		}
+		//		if keyedField["id"] != field.ID {
+		//			t.Errorf("Expected keyed field %v to have id %v got %v", field.Label, field.ID, keyedField["id"])
+		//		}
+		//	}
+		//}
+	}
```

Comment:
@volodymyrZotov I seriously tried to get this unit test to work, but I do not understand enough what is going with these data types.

Could you take it from here at the very least? 🙏🏻